docs/technologies/deno.md

```diff
@@ -0,0 +1,30 @@
+# Deno introduction
+
+Deno is an alternative to node.js. It is a command line tool that can install packages, and execute typescript or wasm. It addditionally has built-in dev tooling around linting, formatting, and testing, removing the need for configuring these manually
+
+# Do I need to install deno to run anode?
+Not directly. This anode repository is a traditional node-based application. It uses traditional nodejs paradigms, (the only difference is using pnpm instead of npm for managing packages).
+
+However, once you launch anode, you'll need to connect it to a runtime agent to execute python code. For example, the given command of `NOTEBOOK_ID=notebook-abc-123 deno run --allow-all --env-file=.env "jsr:@runt/pyodide-runtime-agent"` uses deno for this part of the application. For local development, you'll probably run the runtime agent with deno on the same machine, otherwise it could be run on a completely different computer.
+
+# What kinds of files can you run with deno?
+Deno supports subsets of web standards (think import package), promises, async/await. So, if your package is regular typescript, this will probably work out-of-the box.
+
+However, deno won't run Node code out of the box. If you're writing a deno-native package, this means making a series of source code transformations. So, instead of `import { promises as fs } from 'fs';`, you need to change the code to use the node shim layer of deno, which would be `import * as fs from 'node:fs/promises';`
+
+Similarly, the `npm` package repository is not natively supported. Instead of `import React from react`, you would need to use `import React from npm:react`
```

Comment:
I don't know that I'd say "not natively supported" so much as "requires a prefix". All imports are URLs (unless aliased) to match with the import specification for JavaScript (the standardized version that browsers adhere to).

Reply:
Fair, updated to reflect